### ISSUE TITLE
WCAG 2.4.2 🟠 (Partially Supports): Page Titled

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5146]
**Level:** A
**Principle:** Operable
**Result:** 🟠 Partially Supports

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html)

>[Web pages](https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html#dfn-web-page) have titles that describe topic or purpose.

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.

### Previous audit
#4131 
> ✅ page titled
All pages have titles which reflect their type and content (e.g. home pages have journal name, article pages have author, article and journal information).

 ### Progress since last audit

none

 ### Audit Notes

All pages have titles, there is a default to the journal name as a title in the code. 

Most of hte titles are meaningful, but only within a known context of journal or press site. The titles are not fully descriptive as they do not include the journal/press name, which is particuarly important for pages which appear on press and journal level (e.g. news and contacts). Where this site context does exist, the titles do not follow a consistent pattern. 

 ### Work arising

devise a meaningful pattern for titles that will work across the whole front of house, and then update all title templates.

**Comment by StephDriver** [comment]
## Testing Page Titles
I tested a selection of 45 pages across press and journal pages, this included three different journals in order to ensure journal names were being carried across correctly.  Key results:

### Summary
- ✅ Article Pages.  These follow the pattern `article_title | journal_name`
- ✅ All pages tested had titles
- 🟠 Most static pages had titles that were similar page purpose (e.g. contact) but did not identify the journal or press.
- :x: a handful of pages had titles that were simply the journal name (a default in the code) rather than anything descriptive.


### Specific bugs:
- :x: Special collection pages are missing the collection name, this seems to be a bug as they have the title of `journal_name | Collection:` as if the collection name should follow.
- :x: Clean Issues page has a title of the journal name only
- :x:Clean news items are titled with the news headline only, nothing identifying this as "news"
- :x:  Material and OLH news items are titled `News - news_headline` which is not consistent with the article page pattern.
- :x: OLH journal page has `journal_name` as a title.
